cabot/cabot_config.py

```diff
@@ -5,6 +5,8 @@
 GRAPHITE_PASS = os.environ.get('GRAPHITE_PASS')
 GRAPHITE_FROM = os.getenv('GRAPHITE_FROM', '-10minute')
 JENKINS_API = os.environ.get('JENKINS_API')
+if not JENKINS_API.endswith('/'):
```

Comment:
I think we should fix it in code in both places. Ideally, we should never construct paths ourselves.

For eg: we should use os.path.join( ... ) instead of a + '/' + 'b' + ...
In this case we should use urlpath.urljoin

```
In [5]: urlparse.urljoin('http://booo', 'x/y')
Out[5]: 'http://booo/x/y'

In [6]: urlparse.urljoin('http://booo/', 'x/y')
Out[6]: 'http://booo/x/y'

In [7]: urlparse.urljoin('http://booo/', '/x/y')
Out[7]: 'http://booo/x/y'

In [8]: urlparse.urljoin('http://booo', '/x/y')
Out[8]: 'http://booo/x/y'
```